phoenix-core/src/main/java/org/apache/phoenix/query/DelegateConnectionQueryServices.java

```diff
@@ -361,6 +361,11 @@ public boolean isUpgradeRequired() {
         return getDelegate().isUpgradeRequired();
     }
 
+    @Override
+    public void clearUpgradeRequired() {
+        getDelegate().isUpgradeRequired();
```

Comment:
Shouldn't this be getDelegate.clearUpgradeRequired(); ?

Reply:
Good catch! Copy-paste error. Luckily this method is never really used in our upgrade code path since we use the `CQSI.clearUpgradeRequired`. I will add this to my changes for PHOENIX-5544 since it works with relatively the same path and 5103 is already committed.